wordpress-mobile/WordPress-iOS
# Jetpack connection improvements

**Fixes** #8690 

**To test:**

1. Test that the issue reported in the ticket doesn't happen anymore.
2. Check that the new stats model makes sense: changing to single events with a source property to distinguish between notifications and stats as an origin.

cc: @planarvoid once this is approved please update Android to sync the stats
